docs/JLBP-0022.md

```diff
@@ -0,0 +1,56 @@
+---
+jlbp:
+  id: JLBP-22
+permalink: /JLBP-22
+---
+
+# Declare all dependencies
+
+If your code references a class—for example, by invoking a method in that class—declare a dependency that includes that class in your pom.xml, build.gradle,
```

Comment:
Is there a formatting practice for referring to files? Should this be `pom.xml` or _pom.xml_?

Reply:
I don't think we've chosen one yet. Might be worth checking general google style and general cleanup.